SU2_CFD/src/solvers/CHeatSolver.cpp

```diff
@@ -32,22 +32,17 @@ CHeatSolver::CHeatSolver(void) : CSolver() { }
 
 CHeatSolver::CHeatSolver(CGeometry *geometry, CConfig *config, unsigned short iMesh) : CSolver() {
 
-  unsigned short iVar, iDim, nLineLets, iMarker;
+  flow = ((config->GetKind_Solver() == INC_NAVIER_STOKES) ||
+          (config->GetKind_Solver() == INC_RANS) ||
+          (config->GetKind_Solver() == DISC_ADJ_INC_NAVIER_STOKES) ||
+          (config->GetKind_Solver() == DISC_ADJ_INC_RANS));
 
-  bool multizone = config->GetMultizone_Problem();
-
-  int rank = MASTER_NODE;
-
-  bool heat_equation = ((config->GetKind_Solver() == HEAT_EQUATION) ||
-                        (config->GetKind_Solver() == DISC_ADJ_HEAT));
+  heat_equation = ((config->GetKind_Solver() == HEAT_EQUATION) ||
+                   (config->GetKind_Solver() == DISC_ADJ_HEAT));
```

Comment:
GetFluidProblem, GetHeatProblem ? make the booleans const and use the initializer list to set them forever?

Reply:
One has to set the initializer list for const class-members for all constructors obviously. that cost me a few minutes. I deleted the empty constructor as I couldn't find it being used, i.e. `grep -ri CHeatSolver()` was empty.

Reply:
Concerning`GetFluidProblem`: I guess one could make a `GetIncFluidProblem` to be fully correct, but that should be catched elsewhere.

Reply:
Should not matter for this.